bot/exts/holidays/easter/egghead_quiz.py

```diff
@@ -65,23 +65,43 @@ async def eggquiz(self, ctx: commands.Context) -> None:
 
         msg = await ctx.fetch_message(msg.id)  # Refreshes message
 
-        total_no = sum([len(await r.users().flatten()) for r in msg.reactions]) - len(valid_emojis)  # - bot's reactions
+        users = []
+        for reaction in msg.reactions:
+            async for user in reaction.users():
+                users.append(user)
+
+        total_no = len(users) - len(valid_emojis)  # - bot's reactions
 
         if total_no == 0:
             return await msg.delete()  # To avoid ZeroDivisionError if nobody reacts
 
         results = ["**VOTES:**"]
         for emoji, _ in answers:
-            num = [len(await r.users().flatten()) for r in msg.reactions if str(r.emoji) == emoji][0] - 1
+            users = []
+            for reaction in msg.reactions:
+                if str(reaction.emoji) != emoji:
+                    continue
+
+                async for user in reaction.users():
+                    users.append(user)
+                break
+
+            num = len(users) - 1
             percent = round(100 * num / total_no)
             s = "" if num == 1 else "s"
             string = f"{emoji} - {num} vote{s} ({percent}%)"
             results.append(string)
 
+        users = []
+        for reaction in msg.reactions:
+            if str(reaction.emoji) != correct:
+                continue
+
+            async for user in reaction.users():
+                users.append(user)
+
```

Comment:
Putting a break here like on the last one would make the behaviour more obvious.

Reply:
I don't think a `break` works here, since we *do* want to iterate over all the reactions in this case.

What I will do though, is make `users` a set, since we don't the same user showing up twice.